4/4ti2/build_tarballs.jl

```diff
@@ -0,0 +1,96 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "4ti2"
```

Comment:
I don't think we can have a JLL package with this name, `4ti2_jll` is not a valid Julia identifier

Reply:
That's unfortunate. But I guess I could call it `_4ti2`, which matches what I already do in a few spots in there. If that sounds OK, I'll rename everything accordingly.